### DISCUSSION TITLE
Z Axis improvement by mooving openbuild eccentric to other side

### DISCUSSION THREAD

**Discussion by white777777** [1coderookie/KobraGoNeoInsights#14] [General]
Hi. I've improved z axis by moving eccentric wheel to other side. Here is photo:
![photo_2023-11-04_23-48-00](https://github.com/1coderookie/KobraGoNeoInsights/assets/8267032/908a8c17-4837-4af1-8d76-32beddfcf12b)

Before this improvement my heigmap calibration had show that right side of table a little bit lower than left side. After this change i was able to make z offset equal on left and right side of the table. Heighmap calibration become more repeatable.
<img width="294" alt="Снимок экрана 2023-11-04 235104" src="https://github.com/1coderookie/KobraGoNeoInsights/assets/8267032/6e9d7e6e-f9ba-498d-984f-23483ee156e8">

To do this mod you will need to drill 8mm hole on place of old hole.

PS. Sorry for my english.

**Comment by 1coderookie** [comment]
Hi,
thanks for sharing!  
I like the idea itself, I wouldn't recommend to do so though. The problem here is that by doing so you did eliminate the misalignment of that outer wheel, but you didn't [tram the x-gantry porperly](https://1coderookie.github.io/KobraGoNeoInsights/hardware/axes/#tramming-the-x-axis-gantry).  
The issue with the non-aligning wheels on the outer side at the z-frame rail is due to those Y-shaped brackets (where the wheels mounted to) being mounted slightly crooked. So you need to adjust **their** position and make both outer wheels touch, and then you only adjust the overall pressure of the wheels by adjusting the position of the inner wheel by using that eccentric nut of the inner wheel.  
That's what I tried to illustrate with this picture:  
![axes_gantry_bracket-position_web](https://github.com/1coderookie/KobraGoNeoInsights/assets/25869362/2dfc42ed-1e6b-4017-ad9d-c111ef4a2564)
If you do that tramming properly as described, then there'll be no play left and no (significatnt) sagging of the right side will occur.  
Now that you mounted an eccentric nut there, it'll be much more difficult to actually tram the whole x-gantry properly as that Y-bracket will most likely be in a somewhat crooked position in the end.  
So it's a 'hack' you made that works for getting both outer wheels touching the z-frame, but it would've been better to actually tram the whole setup correctly..
